Client/Frontend/Browser/Favorites/FavoritesViewController.swift

```diff
@@ -359,6 +359,10 @@ extension FavoritesViewController: UICollectionViewDataSource, UICollectionViewD
                 return cell
             }
             
+            cell.setOpenButtonAction { [weak self] in
+                self?.onOpenRecentSearch(recentSearch)
+            }
```

Comment:
`[unowned self]` is probably better here, the `cell` lifetime depends on the VC, so this callback shouldn't ever be hit unless self is alive